src/loaders/lottie/tvgLottieProperty.h

```diff
@@ -502,6 +502,7 @@ struct LottieColorStop : LottieProperty
     Array<LottieScalarFrame<ColorStop>>* frames = nullptr;
     ColorStop value;
     uint16_t count = 0;     //colorstop count
+    uint16_t countOrg = 0;  //originaly declared number of colors in a frame
```

Comment:
```
                "p": 2,    // countOrg
                "k": {
                  "a": 0,
                  "k": [
                    offset, r, g, b,
                    offset, r, g, b,
                    offset, alpha,
                    offset, alpha,
                    offset, alpha
                   ]
```
countOrg is needed to properly read the colorStops, so we need it
but since the colors offset and the alpha offset may be different, the example above may create 5 colorStops or only 3. and we need to know this as well since colors are stored in a memory block, not in an array.

Reply:
> countOrg is needed to properly read the colorStops

@mgrudzinska  Hmm... in your new logic, `countOrg` is only used for population, which is triggered only once for setting. At that point, the value should be exactly the same as `count`. Am I right?

Reply:
sample without the issue with interpolation (fix in https://github.com/thorvg/thorvg/pull/2788)

[colorStops.json](https://github.com/user-attachments/files/17189715/colorStops.json)

populate is called twice (once for each frame):
```
populate:  color.input->count 16  colorStops.countOrg 3  colorStops.count 3
	alphaCnt before pr: 2  after pr: 2 
	 populate  output.count 5  

populate:  color.input->count 16  colorStops.countOrg 3  colorStops.count 5
	alphaCnt before pr: 2147483646  after pr: 2 
	 populate  output.count 5  
```
the result before (checked on dotlottie, thorvg viewer) is not correct:
<img width="400" alt="Zrzut ekranu 2024-09-30 o 14 42 35" src="https://github.com/user-attachments/assets/a00f4770-6986-4cd3-b4dc-e4f3e8225ce7">

after looks correct (same as in skottie and lottie web svg/canvas):
<img width="400" alt="Zrzut ekranu 2024-09-30 o 14 24 39" src="https://github.com/user-attachments/assets/d61fd508-5754-4edf-99d4-a2fe0945ce73">